sys/zuse.hoon

```diff
@@ -678,11 +678,16 @@
     $%  {$bel ~}                                        ::  make a noise
         {$clr ~}                                        ::  clear the screen
         {$hop p/@ud}                                    ::  set cursor position
```

Comment:
Should this maybe be `[x=@ud y=@ud]` now? You can't partially change lines otherwise.